src/lib.rs

```diff
@@ -149,6 +149,8 @@ mod array_serde;
 mod arrayformat;
 mod data_traits;
 
+pub use arrayformat::PRINT_ELEMENTS_LIMIT;
```

Comment:
I'd refer to not reexport this public constant at the top level of the crate, but rather reexport it from some `config` or `integration_test` module.

As far as I can tell the motivation for making it `pub` is entirely for making it usable within the integration tests at `tests/format.rs`?

Reply:
Fixed

Reply:
Awesome. I think that's better. I understand your motivation to have a way of configuring formatting similar to numpy's `set_printoptions`, but we wouldn't be able to actually accomplish that through `const PRINT_ELEMENT_LIMIT`. I am actually not sure right now if that was possible at all for a user of the library to accomplish at compile time.